### PR TITLE
Added missing error response sending to M2MNsdlInterface::received_from_server_callback

### PR DESCRIPTION
**Pull request by simosillankorva**

### CODE REVIEW

**Comment by simosillankorva** [comment]
rebuild

**Comment by simosillankorva** [comment]
rebuild

**Comment by simosillankorva** [comment]
rebuild

File: source/m2mnsdlinterface.cpp [hunk 1]
```diff
@@ -462,24 +462,39 @@ uint8_t M2MNsdlInterface::received_from_server_callback(struct nsdl_s *nsdl_hand
...
+            sn_coap_hdr_s *coap_response = NULL;
+            bool execute_value_updated = false;
+            M2MObjectInstance *obj_instance = NULL;
+
             if(COAP_MSG_CODE_REQUEST_PUT == coap_header->msg_code) {
                 if (is_bootstrap_msg) {
                     handle_bootstrap_put_message(coap_header, address);
                 }
+                else{
+                    tr_debug("Method not allowed (PUT).");
```
Comment (anttiylitokola) [inline source/m2mnsdlinterface.cpp]: debug macro should contain class and function name also.

**Comment by anttiylitokola** [comment]
Small debug print changes otherwise ok.

**Comment by yogpan01** [comment]
rebuild

**Comment by jankii01** [comment]
rebuild

**Comment by simosillankorva** [comment]
rebuild

**Comment by simosillankorva** [comment]
rebuild

**Comment by simosillankorva** [comment]
rebuild

**Pull request merged**